zenstruck/foundry/1.9/config/packages/test/zenstruck_foundry.yaml

```diff
@@ -0,0 +1,2 @@
+imports:
+    - { resource: ../dev/zenstruck_foundry.yaml }
```

Comment:
I would not use an import here, the complexity for one option is not worth.

Any reasons why we should do it this way @kbond ?

Reply:
This was done to avoid users having to duplicate their config in test and dev. There are several other configuration options for foundry and I think the common scenario would be for them to be the same in test/dev.

What do you suggest?

Reply:
What I'm trying to avoid is: when a user decides they need to add additional config, they don't miss adding it in two places (as I believe almost always the user would want the same config for test/dev).

Reply:
Makes sense, thanks for your explanation 👍